### PR TITLE
test: 401 invalid ?include=<path> -> 400 Bad Request

### LINKED ISSUE #87: PUT /session to return 400 Bad Request for invalid ?include=<path>
part of https://github.com/hoodiehq/hoodie-server-account/issues/83

### :octocat:  Your First Pull Request?

We would like to take the opportunity to invite someone who did not yet contribute to Hoodie to work on this issue. Follow [@YourFirstPR](https://twitter.com/yourfirstPR) on Twitter for more of these issues across many open source projects :)

---

### :bug: The Bug

`hoodie-server-account` is implementing a [JSON API](http://jsonapi.org/format/#fetching-includes). The spec says that if passing an invalid path for the `?include=<path>` query parameter, the server must respond with a `400 Bad Request` error. But instead the server currently returns `403 Forbidden`

### :clipboard: Steps to fix

- [x] claim this issue (@c-ro)
- [x] [Setup repository locally](https://github.com/hoodiehq/hoodie-server-account/blob/master/tests/README.md#testing) and make sure that all tests pass
- [x] Adjust the test in [tests/integration/routes/session-test.js](https://github.com/hoodiehq/hoodie-server-account/blob/6b72426/tests/integration/routes/session-test.js#L428-L446) to assert that the server will respond with `400` statusCode and `”Bad Request”` title.
- [x] Make sure tests fail when you run `npm test`. Commit with the message `test: 401 invalid ?include=<path> -> 400 Bad Request`
- [x]  Pull Request #88
- [x] Fix the implementation :)
- [x] Make sure tests pass. Commit with `fix: invalid ?include=<path> -> 400 Bad Request`
- [x] You are done :clap: comment below that your pull request is ready for review 

Ping us in the [Hoodie Chat](http://hood.ie/chat/) or on [Twitter](https://twitter.com/hoodiehq/) if you have any questions :)

### PR DESCRIPTION
**Pull request by c-ro**
test rewritten to reflect spec

closes #87

### CODE REVIEW

File: routes/session.js [hunk 1]
```diff
@@ -118,7 +118,7 @@ function sessionRoutes (server, options, next) {
           })
         }
 
-        throw error
+        // throw error
```
Comment (gr2m) [inline routes/session.js]: why did you comment this out?

Comment (c-ro) [inline routes/session.js]: I likely did that while I was getting my bearings, left in error.  Sorry about that.

Comment (gr2m) [inline routes/session.js]: Coudl remove the change and `git push --force` with the cleaned up commits? If not I can do that for you

Comment (c-ro) [inline routes/session.js]: done.

**Comment by gr2m** [comment]
merged via 76be3f3. I’ve squashed your last two commits into one :)

**Pull request closed without merging**